src/uct/api/uct.h

```diff
@@ -825,6 +825,11 @@ enum uct_md_mem_flags {
      */
     UCT_MD_MEM_ACCESS_LOCAL_WRITE   = UCS_BIT(9),
 
+    /**
+     * Enable using device memory (e.g MEMIC).
```

Comment:
`Allocate device memory. This flag can be specified for @ref uct_mem_alloc and will be used only if the corresponding memory domain supports @ref UCT_MD_FLAG_ALLOC_DEVICE_MEMORY`